### ISSUE TITLE
Ubuntu Build Error: cuda_runtime_api.h not found

### ISSUE THREAD

**Issue by turowicz** [takuya-takeuchi/DlibDotNet#55]
I can build on MacOS with NO cuda.
I get this error on Linux with cuda ON.

As you can see dlib builds fine, so there must be a problem in DlibDotNet.

```
[ 46%] Built target dlib
Scanning dependencies of target DlibDotNet.Native
[ 46%] Building CXX object CMakeFiles/DlibDotNet.Native.dir/dlib/array/array.cpp.o
[ 47%] Building CXX object CMakeFiles/DlibDotNet.Native.dir/dlib/array2d/array2d.cpp.o
[ 48%] Building CXX object CMakeFiles/DlibDotNet.Native.dir/dlib/clustering/chinese_whispers.cpp.o
[ 48%] Building CXX object CMakeFiles/DlibDotNet.Native.dir/dlib/cuda/cuda.cpp.o
In file included from /DlibDotNet/src/DlibDotNet.Native/dlib/cuda/cuda.cpp:1:0:
/DlibDotNet/src/DlibDotNet.Native/dlib/cuda/cuda.h:7:10: fatal error: cuda_runtime_api.h: No such file or directory
 #include "cuda_runtime_api.h"
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/DlibDotNet.Native.dir/dlib/cuda/cuda.cpp.o] Error 1
CMakeFiles/DlibDotNet.Native.dir/build.make:101: recipe for target 'CMakeFiles/DlibDotNet.Native.dir/dlib/cuda/cuda.cpp.o' failed
CMakeFiles/Makefile2:72: recipe for target 'CMakeFiles/DlibDotNet.Native.dir/all' failed
make[1]: *** [CMakeFiles/DlibDotNet.Native.dir/all] Error 2
Makefile:105: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by turowicz** [comment]
@takuya-takeuchi can you help?

**Comment by turowicz** [comment]
Resolved by setting the environment variable `CUDA_PATH=/usr/local/cuda`

**Comment by takuya-takeuchi** [comment]
OK. I tried buding on Linux with CUDA.
But machine did not install CUDA and it fails.

Please give me time to install CUDA and build.

**Comment by turowicz** [comment]
@takuya-takeuchi as said above, the fix was to set `CUDA_PATH` env variable. This post is closed, but I've open another #56 for the incorrect library linked on Linux.
